projects/gateway-utilities/ignition/script-python/TranslationManagementPage/code.py

```diff
@@ -0,0 +1,37 @@
+"""
+General.Translation Management Page
+DESCRIPTION: This page is used to manage translations for a project
+"""
```

Comment:
I think there was some miscommunication about this file. This file should still be located in the General folder with all the other python scripts in this project. The binding on the buttons were not going to the where this file is located.